pkg/3scale/amp/component/system.go

```diff
@@ -555,7 +554,7 @@ func (system *System) volumeNamesForSystemAppPreHookPod() []string {
 	return res
 }
 
-func (system *System) AppDeploymentConfig() *appsv1.DeploymentConfig {
```

Comment:
`priorityClassName` should be passed in Options struct. The usual workflow is: 1) read by the options_provider and 2) populate the options struct.

Reply:
@eguzki , thank you for comment and solution, I tried to do it, meanwhile for 3 components - system, backup and apicast; will continue,

Reply:
Done